packages/analytics/package.json

```diff
@@ -71,5 +71,8 @@
       ".ts"
     ],
     "reportDir": "./coverage/node"
+  },
+  "engines": {
+    "node": ">=18.0.0"
```

Comment:
The spreadsheet notes that analytics and some of the types packages aren't supported in node. I had assumed that we wouldn't see a node engines entry in those packages' package.json files..?

Reply:
Yep, looks like the script didn't work, I'll fix it.

Reply:
I updated the PR and also made a gist of the array in my script telling the script which packages to add this to, to make it easier for you to audit than having to go through all the files in this PR https://gist.github.com/hsubox76/e58e8a45d1cb04af7ebb31ebff07365d Let me know if any of them look incorrect.